bundles/core/org.eclipse.smarthome.core.thing.xml.test/build.properties

```diff
@@ -1,4 +1,5 @@
 source.. = src/test/groovy/,\
+		   src/test/java/,\

```

Comment:
is `src/test/groovy` still valid? It looks to me like no groovy file is remaining, therefore I'd expect the builder to complain here.

Kudos btw for migrating them all!

Reply:
obsolete, indeed.